antora-playbook.yml

```diff
@@ -26,7 +26,7 @@ content:
       branches: [ master ]
       start_path: docs
     - url: https://github.com/quarkiverse/quarkus-cxf.git
-      branches: [ master ]
+      branches: [master, v*]
```

Comment:
You can also use `HEAD` instead of `master` in case the branch gets renamed

Reply:
Whoops my bad, just saw this comment.  Did you want me to address it?  I think master is OK for now, though if needed I can switch it to HEAD later, if that works?

Reply:
That's fine, I might probably send a separate PR changing all of them to `HEAD` at some point